Tools/WebKitTestRunner/TestOptions.cpp

```diff
@@ -92,6 +92,7 @@ const TestFeatures& TestOptions::defaults()
             { "DataTransferItemsEnabled", true },
             { "DeveloperExtrasEnabled", true },
             { "DirectoryUploadEnabled", true },
+            { "DisallowSystemFontFallbackForPrivateUseAreaCharacters", true },
```

Comment:
Do we need this for DumpRenderTree, as well? I think we do so that WK1 tests will work.